### PR TITLE
Translate Ruby 2.5.0-rc1 Released (pt)

### PR DESCRIPTION
**Pull request by jcserracampos**

### CODE REVIEW

**Comment by stomar** [comment]
trailing whitespace

**Comment by jcserracampos** [comment]
@stomar Trimming trailing whitespace is a recurrent problem while using Code.
I will see what I can do.

File: pt/news/_posts/2017-12-14-ruby-2-5-0-rc1-released.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+* rescue/else/ensure passam a ser permitidos em blocos do/end.  [Feature #12906]
+
+* Adiciona yield\_self.  [Feature #6721]
+
+## Melhorias de performance
+
+* Instrumentação dinâmica para hooks TracePoint ao invés de usar instrução
+  "trace" para evitar sobrecarga. [Feature #14104]
+
+* Performance de passagem de bloco usando parâmetros de blocos
```
Comment (JuanitoFatas) [inline pt/news/_posts/2017-12-14-ruby-2-5-0-rc1-released.md]: There is another trailing space after `blocos`.

Comment (jcserracampos) [reply pt/news/_posts/2017-12-14-ruby-2-5-0-rc1-released.md]: I will be very thankful to you.
My code is beating me.

Comment (JuanitoFatas) [reply pt/news/_posts/2017-12-14-ruby-2-5-0-rc1-released.md]: Could you add me to `https://github.com/jcserracampos/www.ruby-lang.org` 😊 
Then I will push the change in this PR.

Comment (jcserracampos) [reply pt/news/_posts/2017-12-14-ruby-2-5-0-rc1-released.md]: Done!

**Comment by JuanitoFatas** [comment]
@jcserracampos thanks for your contribution! (by the way, you could open a branch on ruby/www.ruby-lang.org instead of working on your fork), Cheers.

**Comment by jcserracampos** [comment]
@JuanitoFatas Thanks. You are awesome.

**Comment by jcserracampos** [comment]
@JuanitoFatas I do not have writing permission here.
I tried to update a branch.

**Pull request merged**